src/backend/utils/misc/queryenvironment.c

```diff
@@ -384,6 +285,16 @@ get_ENR_withoid(QueryEnvironment *queryEnv, Oid id, EphemeralNameRelationType ty
 	return NULL;
 }
 
+/*
+ * This is a wrapper function to get_ENR_withoid with the commonly 
+ * provided arguments of currentQueryEnv and ENR_TSQL_TEMP.
+ */
+EphemeralNamedRelation
+OidBelongsToENRTempTable(Oid id)
+{
```

Comment:
Do we really need this? seems like it's a one line function so we can just call `get_ENR_withoid` directly.
ALso, the function name suggests it's supposed to return true/false but it's returning a `EphemeralNamedRelation`

Reply:
My thinking was just to add this for readability, since `get_ENR_withoid` had some arguments that never change. For instance, it's almost always called with `ENR_TSQL_TEMP` as the third argument. Maybe we can just rewrite the base function instead.

Reply:
I see your point. It makes sense to rename `OidBelongsToENRTempTable` to something more descriptive then because I initially thought the name would return true/false. Something like `GetENRTempTableWithOid(Oid oid)` or something better (im not good in namings) :)

Reply:
+1 on renaming this function to something more accurate to its behavior.

Reply:
that name LGTM! refactored.